CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.35.0 (July 27th, 2022)
+
+BUG FIXES:
+* Added `Computed` modifier to attributes in `r/tfe_registry_modules` to prevent unnecessary registry modules update. ([#572](https://github.com/hashicorp/terraform-provider-tfe/pull/572))
```

Comment:
```suggestion
* r/tfe_registry_module: Added `Computed` modifier to attributes in order to prevent unnecessary resource replacement ([#572](https://github.com/hashicorp/terraform-provider-tfe/pull/572))
```